pages.pt_BR/windows/title.md

```diff
@@ -0,0 +1,8 @@
+# title
+
+> Exibe o título do prompt de comando.
```

Comment:
It seems to me that the command description and the examples are incoherent: 
In the description, we say it shows the title, and we only have an example to change the prompt title. 
Perhaps it would be good to also add an example on how to show the prompt title. This would need to be done on both files: English and Portuguese. Perhaps you want to tackle this? 

Since this is a translation request I will approve it as this is not a translation problem.